tests/containers/rootless_podman.pm

```diff
@@ -59,8 +57,21 @@ sub run {
         verify_userid_on_container($runtime, $iname);
     }
     clean_container_host(runtime => $runtime);
+}
+
+sub post_run_hook {
+    my $self = shift;
+    record_info "post-cleanup";
```

Comment:
this looks confusing in VR https://openqa.suse.de/tests/5502787#step/rootless_podman/21 . empty "post-cleanup" record-info and  than nothing afterwards . I would  drop it at all ( you not showing anything when you changing acl so no point to highlight revert ). If you want to keep it than add more info so it will be clear what is happening

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/opensusebasetest.pm#L53 clears the console, and then we will pollute it with the ACL revet action. 
Maybe we should swap these 2 actions. Personally, I would also drop tty/console selection over here if possible.

Reply:
@asmorodskyi removed
@mloviska done